src/components/App/index.js

```diff
@@ -43,40 +46,53 @@ const App = () => {
     setExportHref(getExportHrefValue(playList))
   }, [playList])
 
+  useEffect(() => {
+    const root = window.document.documentElement
+    root.classList.remove(colorTheme === 'dark' ? 'light' : 'dark')
+    root.classList.add(colorTheme)
+    localStorage.setItem('theme', colorTheme)
+  }, [colorTheme])
+
   return html`
-    <div class="p-4">
+    <div class="p-4 dark:bg-[#18181b] h-screen transition duration-500">
```

Comment:
Apparently the `h-screen` class is generating a bug:
![image](https://user-images.githubusercontent.com/30677819/171034236-194716e2-1590-4a61-99cd-54984b5920bb.png)

Reply:
You're right. In order to resolve this I need to add a classs (or apply css rules) to the div with id "app". Do u know how can I do this? Because I see this div is rendered like a component but I don't understand where I can put/add some classes

Reply:
because otherwise the page content does not fill the entire page, and in dark mode there will be some white. This is an example without h-screen:

![immagine](https://user-images.githubusercontent.com/41837640/171272141-caab5715-92d6-43b2-a70c-3e69e7c2f63d.png)

Reply:
Yes, you're right too. I did some tests and looks like changing the `h-screen` to `min-h-screen` solved the problem. Can you do this test

Reply:
Yes it worked. Thanks!